src/zig_clang_driver.cpp

```diff
@@ -558,7 +558,7 @@ int ZigClang_main(int argc_, const char **argv_) {
   if (!UseNewCC1Process && IsCrash) {
     // When crashing in -fintegrated-cc1 mode, bury the timer pointers, because
     // the internal linked list might point to already released stack frames.
-    llvm::BuryPointer(llvm::TimerGroup::aquireDefaultGroup());
+    llvm::BuryPointer(llvm::TimerGroup::acquireDefaultGroup());
```

Comment:
I need to drop this. llvm has been on my todo list for a long time, but I haven't gotten around to it.